### PR TITLE
930 views ui admin consistency

### PR DESCRIPTION
**Pull request by wesruv**
Fixes https://github.com/backdrop/backdrop-issues/issues/930.

Here's a screenshot of the views landing page: 
screen shot 2017-04-27 at 7 54 14 pm

### CODE REVIEW

**Comment by wesruv** [comment]
Here's what the admin/content View looks like when being edited:
image

File: core/modules/views_ui/css/views_ui.admin.theme.css [hunk 1]
```diff
@@ -681,7 +681,7 @@ ul#views-display-menu-tabs li.add ul.action-list li{
 
 #views-ui-preview-form .arguments-preview,
 #views-ui-preview-form .form-type-textfield {
-  margin-left: 14px;
+  margin: 0 0 0 14px;
```
Comment (quicksketch) [inline core/modules/views_ui/css/views_ui.admin.theme.css:684]: FYI this looks like it could use a matching `[dir="rtl"]` rule.

File: core/modules/views_ui/css/views_ui.admin.theme.css [hunk 2]
```diff
@@ -49,7 +49,7 @@
     url("../images/sprites.png"),
     -moz-linear-gradient(
       -90deg,
-      #ffffff 0px,
```
Comment (jenlampton) [inline core/modules/views_ui/css/views_ui.admin.theme.css:52]: Were these changes intentional? (or perhaps automatically changed by an editor?)  I think our standard is to show all 6 characters of a hex code.

File: core/modules/views_ui/css/views_ui.admin.theme.css (hunk 1 above)
Comment (jenlampton) [reply core/modules/views_ui/css/views_ui.admin.theme.css:684]: looks like this has been added.

**Comment by jenlampton** [comment]
Replaced by https://github.com/backdrop/backdrop/pull/1846
